### DISCUSSION TITLE
How to get page id from the url ?

### DISCUSSION THREAD

**Discussion by aahnik** [ramnes/notion-sdk-py#31] [Q&A] [answered]
Is the end part of a page its id ?

for example I have an URL

```text
https://www.notion.so/aahnik/notion-sdk-py-540f8e2b79914654ba103c5d8a03e10e
```

that means, the id is

```text
540f8e2b79914654ba103c5d8a03e10e
```

But that does not seem to work. I mean that the end part of the URL slug is not the page id. I thought it to be so.

Is there any quick way to obtain a page id from its URL, without writing any code?

**Comment by Blucknote** [comment]
What do you mean not working? Did you [shared](https://developers.notion.com/docs/getting-started#share-a-database-with-your-integration) page with your integration?

**Reply by aahnik** [reply]
> What do you mean not working?

I mean that the end part of the URL slug is not the page id. I thought it to be so.

**Reply by aahnik** [reply]
Hi @Blucknote thats the right answer. Thanks a lot. But please delete it as a reply, and post as an answer, so that it can be marked as answer.

also edit the link of the Doc reference, its invalid. (append https:// in front of it )

**Reply by Blucknote** [reply]
This new GitHub features are strange o_O

**Reply by Blucknote** [reply]
Oh. This is Discussion. Didnt notice that

**Comment by Blucknote** [comment, marked as answer]
It is page or database UUID (id)


```
https://www.notion.so/{workspace_name}/{database_id}?v={view_id}
```

[Doc refference](https://developers.notion.com/docs/working-with-page-content)
![image](https://user-images.githubusercontent.com/15163043/118937122-d7dee580-b955-11eb-9d1f-2216f4c197ac.png)
